chapter2/tests/t6/chapter2-l2-t6-get-file-unique-dirs.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Task 6 - get only dirs of the files
+set -eo pipefail
+
+
+# Function for printing error messages and exiting with a specific exit code
+# $1 - The exit code to use when exiting the script
+# $2 - The error message to print
+err() {
+  local code="$1"
+  shift
+  echo "[[ERROR]: $(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
+  exit "$code"
+}
+
+
+# Args check
+if [ "$#" -ne 1 ]; then
+  err 255 'Two arguments are required.'
```

Comment:
2 arguments? Looks like this is incorrect

Reply:
Sure